packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,204 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "PIR Control",
+			"maxNodes": 4
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Basic Set Value",
+			"description": "Set the value of Basic Set commands sent to the devices associated in Group 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"$if": "firmwareVersion >= 2",
```

Comment:
Yes, like I reviewed above :)

Reply:
@NYZack didn't you say you changed this?

Reply:
@AlCalzone and @blhoward2 Not sure I understand. If you're asking why I didn't include conditional parameters for firmware < 2.0, it's because I can't find documentation of the precise details (looked in OZW, Smartthings, OpenHab).  I initially included some conditional parameters for firmware 1.03, but I wasn't confident that the details were correct and thought it better to leave them out.

Basically, there was a hardware version 1, with firmware 1.0 - this had no configuration parameters.  
HW version 1 was updated to FW 1.03 that included some configuration parameters, but I can't find documentation of the details of these (in particular, what the parameter numbers were).
HW version 2 included firmware versions 2.0 and 2.01 (which  have the same configuration parameters as each other) - these are well-documented and included here.

Sorry for the million commits. This is new to me.  (And sorry if I'm not understanding your question.)

Reply:
```suggestion
			"$if": "firmwareVersion >= 2.0",
```
I meant this ^

Reply:
That was easy.  Done.